abci/proposals.go

```diff
@@ -57,12 +54,24 @@ type (
 	// ProposalHandler contains the functionality and handlers required to\
 	// process, validate and build blocks.
 	ProposalHandler struct {
+		logger    log.Logger
+		txEncoder sdk.TxEncoder
+		txDecoder sdk.TxDecoder
+
+		// prepareLanesHandler is responsible for preparing the proposal by selecting
+		// transactions from each lane according to each lane's selection logic.
 		prepareLanesHandler blockbuster.PrepareLanesHandler
+
+		// processLanesHandler is responsible for verifying that the proposal is valid
+		// according to each lane's verification logic.
 		processLanesHandler blockbuster.ProcessLanesHandler
-		tobLane             TOBLaneProposal
-		logger              log.Logger
-		txEncoder           sdk.TxEncoder
-		txDecoder           sdk.TxDecoder
+
+		// tobLane is the top of block lane which is utilized to verify transactions that
+		// should be included in the top of block.
+		tobLane TOBLaneProposal
```

Comment:
discussed offline, was wondering why the tobLane exists independently of the other lanes